### DISCUSSION TITLE
Dose smplayer support open file  in terminal?

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [smplayer-dev/smplayer#925] [General]
![image](https://github.com/smplayer-dev/smplayer/assets/30348075/aef53acf-5716-47ff-9560-1997ccdc1d84)
it work in linux, but it doesn't work in windows

**Comment by smplayer-dev** [comment]
It should work. Maybe it's because of the Chinese characters.

**Comment by DreamMaoMao** [comment]
https://github.com/smplayer-dev/smplayer/assets/30348075/76b2851a-83d1-4ace-acb9-d880a805b828

but it work in linux  with the same path name.

**Comment by smplayer-dev** [comment]
Please send the smplayer log (menu View) after triyng to play the file on Windows.

**Comment by DreamMaoMao** [comment]
I checked the mpv log and found that the path arg was passed to mpv, then I tried to use the absolute path instead of the relative path, and the problem was solved, thanks for you help
